packages/core/components/Modal/MoveFileManifest/index.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import { useSelector } from "react-redux";
+
+import { ModalProps } from "..";
+import BaseModal from "../BaseModal";
+import { PrimaryButton } from "../../Buttons";
+import { interaction, selection } from "../../../state";
+import FileDetail from "../../../entity/FileDetail";
+import styles from "./MoveFileManifest.module.css";
+import FileSelection from "../../../entity/FileSelection";
+
+/**
+ * Modal overlay for displaying details of selected files for NAS cache operations.
+ */
+export default function MoveFileManifest({ onDismiss }: ModalProps) {
+    // const dispatch = useDispatch(); //TODO: add onMove functionality
```

Comment:
I think this PR should/can include the action that will eventually contain the call to FSS. What I mean is that you can dispatch a "moveFIles" action that you can create in the interaction state branch & create an empty "logics" in the logic middleware file that will just log to the console rather than call FSS for now

Reply:
resolved in [e159438](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/e1594389fa5dbe6648cf9d9208c74ad9ea5fbbf3) and [8c810a6](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/8c810a631d8892a318daba8284aa6585d91952d0)